packages/dialog/src/DialogBase.ts

```diff
@@ -22,10 +22,10 @@ import { property } from '@spectrum-web-components/base/src/decorators.js';
 import '@spectrum-web-components/underlay/sp-underlay.js';
 import '@spectrum-web-components/button/sp-button.js';
 
-import '../sp-dialog.js';
+import '@spectrum-web-components/dialog/sp-dialog.js'
```

Comment:
Please comment the use case (and a little about why it'll actually work, to save future confusion) for each of the changes registering paths.

Reply:
Can you add something small inline. Something like `// Leveraged in build systems that use aliasing to prevent multiple registrations: https://github.com/adobe/spectrum-web-components/pull/3225` or so?

Reply:
Updated.